### ISSUE TITLE
Unnecessary stdin check for taxonkit filter?

### ISSUE THREAD

**Issue by standage** [shenwei356/taxonkit#36]
In TaxonKit 0.7, I get the following error.

```
$ taxonkit filter --list-order
23:31:23.142 [ERRO] stdin not detected
```

The command is checking for stdin, even when input is not used. Can fix by echoing an empty string, but that seems unnecessary.

```
$ echo '' | taxonkit filter --list-order | tail
species
forma specialis,pathovar,subspecies
pathogroup,serogroup
biotype,genotype,serotype
aberration,morph,varietas,variety
subaberration,submorph,subvarietas,subvariety
form,forma
subform,subforma
strain
isolate
```

The same is true for `taxonkit filter --list-rank`.

----------

### Prerequisites

- [x] make sure you're are using the latest version by `taxonkit version`
- [x] read the [usage](http://bioinf.shenwei.me/taxonkit/usage/)

### Describe your issue

* [x] describe the problem
* [x] provide a reproducible example

**Comment by shenwei356** [comment]
Oh, sorry, I fix this.

**Comment by shenwei356** [comment]
Should I release a new version?

- [taxonkit_darwin_amd64.tar.gz](https://github.com/shenwei356/taxonkit/files/5853649/taxonkit_darwin_amd64.tar.gz)
- [taxonkit_linux_amd64.tar.gz](https://github.com/shenwei356/taxonkit/files/5853651/taxonkit_linux_amd64.tar.gz)

**Comment by standage** [comment]
Wow, that was fast!

> Should I release a new version?

Maybe wait? I will continue updating bindings tomorrow, and I may encounter other questions.

**Comment by standage** [comment]
> Maybe wait? I will continue updating bindings tomorrow, and I may encounter other questions.

Ok, with recent fixes in #36 and #37 I haven't found any other bugs or questions about software behavior. You're welcome to release a new version whenever it's convenient for you.

Thanks for your excellent responses!
